paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -998,6 +999,50 @@ internal class PaymentSheetViewModelTest {
         )
     }
 
+    @Test
+    fun `Sends correct event when navigating to AddFirstPaymentMethod screen`() = runTest {
+        val viewModel = createViewModel(
+            stripeIntent = PaymentIntentFixtures.PI_OFF_SESSION,
+        )
+        viewModel.savedStateHandle[SAVE_GOOGLE_PAY_STATE] = GooglePayState.Available
+
+        val receiver = viewModel.currentScreen.testIn(this)
+        viewModel.transitionToFirstScreen()
+
+        verify(eventReporter).onShowNewPaymentOptionForm(
+            linkEnabled = eq(viewModel.isLinkEnabled.value!!),
```

Comment:
I typically prefer explicit values in assertions. Would be nice to have an explicit true/false here.

I also think you need to rebase after my latest merge.

Reply:
Done! Also added two more tests to make sure that the parameters are reported correctly.